product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -50,7 +50,7 @@ A measure of availability expressed as a percentage of uptime in a given year. T
 
 #### Node
 
-One database server in a cluster. A term "node" differs from the term "database server" because there is more than one node in a cluster. A node includes the database server, the OS, and the physical hardware, which is always separate from other nodes in a high-availability context.
+One database server in a cluster. A term *node* differs from the term *database server* because there's more than one node in a cluster. A node includes the database server, the OS, and the physical hardware, which is always separate from other nodes in a high-availability context.
```

Comment:
Not sure why this is switched to a bold treatment. Quoting is perfectly acceptable as its referring to phrases or words and their context.

Reply:
It's actually italic, which is what we use to introduce a term. Quotes are technically (also) correct here but we don't have any standard that says when to use them, and I don't have any precedent in my editing background for using them in tech doc. So in general I'd prefer we stay away from them.